pkg/shared/accountmgmtutil/ams.go

```diff
@@ -289,3 +346,15 @@ func unique(s []string) []string {
 	}
 	return result
 }
+
+func uniqueQuota(s []QuotaSpec) []QuotaSpec {
+	inResult := make(map[QuotaSpec]bool)
+	var result []QuotaSpec
+	for _, quota := range s {
+		if _, ok := inResult[quota]; !ok {
+			inResult[quota] = true
+			result = append(result, quota)
+		}
+	}
+	return result
+}
```

Comment:
This is a small thing but I had to read this a few times to figure out what was happening (we are removing all duplicates from the slice). Maybe just giving this a rename would make it clearer for code reading in the future.

Reply:
Something like, removeDuplicateQuotaSpec maybe?

Reply:
Having unique as prefix makes more sense imo, I have added an inline description for the method. Can you check it and let me know wdyt?

Reply:
Yeah, the comment is fine, I tend to rather document with the naming of things then comments as they can be outdated and not read most of the time.

I have a small rule, if my comment isnt 3 or more lines long it probably means the code can be made clearer without it (or there is something very niche happening that is just difficult so its fine). 

But that is just how I go about it so this is fine 👍